tests/rtl/hardware_test.py

```diff
@@ -340,6 +341,44 @@ def test_hardware_add42_single():
 
     return sdfg
 
+@xilinx_test()
+def test_hardware_axpy_double_pump():
+    # Grab the double pumped AXPY implementation the samples directory
+    spec = importlib.util.spec_from_file_location(
+        "axpy",
+        Path(__file__).parent.parent.parent / "samples" / "rtl" / "axpy_double_pump.py")
+    axpy = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(axpy)
+
+    # Configuration has to use multiple clocks in this sample.
+    old_freq = dace.config.Config.get('compiler', 'xilinx', 'frequency')
+    dace.config.Config.set('compiler', 'xilinx', 'frequency', value='"0:300\\|1:600"')
+
+    # init data structures
+    N = dace.symbol('N')
+    N.set(16)
+    a = np.float32(0)  #np.random.rand(1)[0].astype(np.float32)
+    x = np.random.rand(N.get()).astype(np.float32)
+    y = np.random.rand(N.get()).astype(np.float32)
+    result = np.zeros((N.get(), )).astype(np.float32)
+
+    # Build the SDFG
+    sdfg = axpy.make_sdfg()
```

Comment:
I would also test with another vect. width

Reply:
This has been implemented.